workspace/workflows/jobs.py

```diff
@@ -220,24 +215,28 @@ def summarise_all(branch):
 
 
 def summarise_org(org, branch):
-    blocks = [
-        get_header_block(f"Workflows for {org} repos"),
-        get_text_block(RepoWorkflowReporter.EMOJI_KEY),
-    ]
+    blocks = [get_header_block(f"Workflows for {org} repos")]
     for repo in config.REPOS[org]:
         blocks.append(RepoWorkflowReporter(org, repo, branch).summarise())
     return json.dumps(blocks)
 
 
 def _get_command_line_args():  # pragma: no cover
     parser = argparse.ArgumentParser()
-    parser.add_argument("--target", required=True)
+    parser.add_argument("--target")
     parser.add_argument("--branch", default="main")
+    parser.add_argument("--key", action="store_true", default=False)
```

Comment:
You could use [subparsers](https://docs.python.org/3/library/argparse.html#sub-commands
) here 

I think it'd make this function look something like:
```
# create the top-level parser
parser = argparse.ArgumentParser()
subparsers = parser.add_subparsers(required=True)

# create the parser for the "show" command
show_parser = subparsers.add_parser('show')
show_parser.add_argument('--target', required=True)
show_parser.add_argument('--branch', default="main")
show_parser.set_defaults(func=main)

# create the parser for the "key" command
key_parser = subparsers.add_parser('key')
key_parser.set_defaults(func=get_text_block_for_key)

# parse the args
args = parser.parse_args()
```

And then call (and print, for our purposes) whatever function was selected:
```
print(args.func(args))
``` 
It'd mean you'd need to return the args without calling `var()` on them, and let `main()` handle the splitting of `target`

Reply:
Definitely worth doing for clarity - will pick this up in a separate PR